modules/local/run_af2.nf

```diff
@@ -40,7 +41,7 @@ process RUN_AF2 {
         model_preset = model_preset + " --pdb70_database_path=./pdb70/pdb70_from_mmcif_200916/pdb70 "
     }
     """
-    if [ -d pdb_seqres/pdb_seqres.txt ]; then sed -i "/^\\w*0/d" pdb_seqres/pdb_seqres.txt; fi
+    if [ -d pdb_seqres ]; then sed -i "/^\\w*0/d" pdb_seqres/pdb_seqres.txt; fi
```

Comment:
I think this will fail if the file is actually not in the path, maybe you assume it is but...
Again for readability maybe I will avoid having a oneliner and would instead have something such as:
```suggestion
    if [ -f pdb_seqres/pdb_seqres.txt ]
        then sed -i "/^\\w*0/d" pdb_seqres/pdb_seqres.txt
    fi
```
There are no guidelines for the oneliner so this is up to you

Reply:
Great thanks!